EPlast/EPlast.Tests/Services/PDF/PDFServiceTests.cs

```diff
@@ -0,0 +1,139 @@
+                .Returns(GetTestUserProfilesQueryable());
+            _repository.Setup(rep => rep.CityMembers.FindByCondition(It.IsAny<Expression<Func<CityMembers, bool>>>()))
+                .Returns(GetTestCityMembersQueryable());
+            _repository.Setup(rep => rep.ClubMembers.FindByCondition(It.IsAny<Expression<Func<ClubMembers, bool>>>()))
+                .Returns(GetTestClubMembersQueryable());
+            _logger.Setup(log => log.LogError(It.IsAny<string>()));
+            _pdfService = new PdfService(_repository.Object,_logger.Object,_decisionBlobStorage.Object);
+        }
+
+        [TestCase(1)]
+        [TestCase(546546)]
+        public void DecisionCreatePDFAsync_ReturnsByteArray_Test(int decisionId)
+        {
+
+            _repository.Setup(rep => rep.Decesion.GetFirstAsync(It.IsAny<Expression<Func<Decesion, bool>>>(),
+                    It.IsAny<Func<IQueryable<Decesion>, IIncludableQueryable<Decesion, object>>>()))
+                .ReturnsAsync(Decesions.FirstOrDefault());
+            _decisionBlobStorage.Setup(blob => blob.GetBlobBase64Async(It.IsAny<string>())).ReturnsAsync("Blank");
+           
+            var actualReturn = _pdfService.DecisionCreatePDFAsync(decisionId);
+
+            _repository.Verify(rep => rep.Decesion.GetFirstAsync(It.IsAny<Expression<Func<Decesion, bool>>>(),
+                It.IsAny<Func<IQueryable<Decesion>, IIncludableQueryable<Decesion, object>>>()), Times.Once);
+            Assert.IsInstanceOf<byte[]>(actualReturn.Result);
+        }
+        [TestCase("1")] 
+        [TestCase("546546")]
+        public void BlankCreatePdfAsync_ReturnsByteArray_Test(string userId)
+        {
+            var actualReturn = _pdfService.BlankCreatePDFAsync(userId);
+            _repository.Verify();
+            Assert.IsInstanceOf<byte[]>(actualReturn.Result);
+        }
+
+        [TestCase("1")]
+        [TestCase("546546")]
+        public void BlankCreatePdfAsync_ReturnsNull_Test(string userId)
+        {
+            _repository.Setup(rep => rep.CityMembers.FindByCondition(It.IsAny<Expression<Func<CityMembers, bool>>>()))
+                .Returns(new List<CityMembers>().AsQueryable());
+
+            var actualReturn = _pdfService.BlankCreatePDFAsync(userId);
+            _logger.Verify();
+            Assert.AreEqual(null,actualReturn.Result);
+        }
+
+        [TestCase(1)]
+        [TestCase(546546)]
+        public void DecisionCreatePDFAsync_ReturnsNull_Test(int decisionId)
+        {
+            var decisionRepository = new Mock<IDecesionRepository>();
+            _repository.Setup(rep => rep.Decesion).Returns(decisionRepository.Object);
+            var actualReturn = _pdfService.DecisionCreatePDFAsync(decisionId);
+            _repository.Verify(rep => rep.Decesion, Times.Once);
+            _logger.Verify();
+            Assert.AreEqual(null, actualReturn.Result);
```

Comment:
```suggestion
            Assert.Null(actualReturn.Result);
```
it seems to me that this class has such method for check

Reply:
Fixed it, thanks.